### DISCUSSION TITLE
How to work with multiple fragment files?

### DISCUSSION THREAD

**Discussion by xgaoo** [aertslab/scenicplus#229] [Q&A] [answered]
I have multiple samples and thus multiple fragments.tsv.gz files. How to use them together to infer the GRN? In the tutorial, it seems only one file is allowed for one GRN.
fragments_dict = {'10x_pbmc': os.path.join(work_dir, 'data/pbmc_granulocyte_sorted_3k_atac_fragments.tsv.gz')}

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @xgaoo 

You can use multiple samples. See: https://pycistopic.readthedocs.io/en/latest/Cortex_pycisTopic.html for examples.

Best,

Seppe

**Reply by xgaoo** [reply]
Thank you Seppe. I will try to generate one GRN with multiple samples.
